SwiftScripts/Sources/SchemaDownload/main.swift

```diff
@@ -20,10 +20,20 @@ let output = sourceRootURL
     .appendingPathComponent("Sources")
     .appendingPathComponent("UploadAPI")
 
+// Introspection download:
 let options = ApolloSchemaOptions(schemaFileName: "schema",
-                                  endpointURL: endpoint,
+                                  downloadMethod: .introspection(endpointURL: endpoint),
                                   outputFolderURL: output)
 
+// Registry download:
```

Comment:
This is left commented out for end to end testing in the future - not a great way to end to end test this without setting up a whole way of getting secrets into the repo.